hw/xbox/nv2a.c

```diff
@@ -4496,6 +4674,11 @@ static void pgraph_method(NV2AState *d,
         pg->constants[59].dirty = true;
         break;
 
+    case NV097_SET_EYE_POSITION ...
+            NV097_SET_EYE_POSITION + 12:
```

Comment:
+12 implies DWORD offsets 0, 4, 8 and 12 are handled here. But, is eye a vec4 or a vec3? In case of vec3, the fourth write could corrupt another register...